site/docs/infrastructure-damaged.mdx

```diff
@@ -0,0 +1,56 @@
+---
+description: Daily reports on settlers' attacks on the Palestinian infrastructure, resulting in a parial or total damage. Useful applications include time series, trend analysis, timelines.
```

Comment:
We should note that these numbers are specifically for Gaza so I don't think `settlers` is the right reference here since that's more of a West Bank issue.

Maybe:

```
Daily reports on the impact of IDF attacks on civilian infrastructure in Gaza, resulting in a partial or total damage. Useful applications include time series, trend analysis, timelines.
```